Makefile

```diff
@@ -12,6 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+.PHONY: build
+build: bin/cmrel
+
+.PHONY: bin/cmrel
+bin/cmrel:
+	go build -o $@ ./cmd/cmrel
```

Comment:
note: this doesn't need to be .PHONY but I wanted to keep the changes super minimal for now